### PR TITLE
Use command framework for contact printing action

### PR DESCRIPTION
**Pull request by d-roesch**
This pull request continues the concept of https://github.com/elexis/elexis-3-core/commit/1c9c2bb5ddb49975d9857184260712420ee300e9

### CODE REVIEW

File: bundles/ch.elexis.core.ui/plugin.xml [hunk 1]
```diff
@@ -776,6 +776,12 @@
             id="ch.elexis.core.ui.commands.printAddressLabel"
             name="%elexis.printAddresslabel">
       </command>
+      <command
+            categoryId="ch.elexis.core.ui.commands.kategorie"
+            defaultHandler="ch.elexis.core.ui.commands.PrintContactLabelHandler"
+            id="ch.elexis.core.ui.commands.printContactLabel"
+            name="%elexis.printContactLabel">
```
Comment (col-panic) [inline bundles/ch.elexis.core.ui/plugin.xml]: Can't find this string in plugin.properties

File: bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 2]
```diff
@@ -406,28 +406,16 @@ public void doRun(){
...
 			@Override
 			public void run(){
-				Kontakt kontakt = (Kontakt) ElexisEventDispatcher.getSelected(Kontakt.class);
-				if (kontakt == null) {
-					SWTHelper.showInfo("Kein Kontakt ausgewählt",
-						"Bitte wählen Sie vor dem Drucken einen Kontakt!");
-					return;
+				Command cmd = cmdService.getCommand("ch.elexis.core.ui.commands.printContactLabel");
+				if (!cmd.isDefined()) {
+					  cmd.define(Messages.GlobalActions_PrintContactLabel, Messages.GlobalActions_PrintContactLabelToolTip, cmdCategory);
```
Comment (col-panic) [inline bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: why is this necessary? if it is not defined and you would manually define it here - where is the default handler defined?

File: bundles/ch.elexis.core.ui/plugin.xml (hunk 1 above)
Comment (d-roesch) [reply bundles/ch.elexis.core.ui/plugin.xml]: Fixed in a38492ea7a255b9332e9f0f7a5be685064fda861

File: bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java (hunk 2 above)
Comment (d-roesch) [reply bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: Fixed. The checks are unnecessary because all commands are well-defined in plugin.xml. The default handlers are also defined in plugin.xml.
